assets/js/components/DashboardNavigation.js

```diff
@@ -99,76 +98,129 @@ export default function DashboardNavigation() {
 
 	const breakpoint = useBreakpoint();
 
-	const [ selectedIds, setSelectedIds ] = useState( [
-		global.location.hash.substr( 1 ),
-	] );
+	const [ selectedID, setSelectedID ] = useState(
+		global.location.hash.substring( 1 )
+	);
 
 	const handleSelect = useCallback(
-		( selections ) => {
-			const [ hash ] = selections;
-			if ( hash ) {
-				global.history.replaceState( {}, '', `#${ hash }` );
-
-				global.scrollTo( {
-					top:
-						hash !== ANCHOR_ID_TRAFFIC
-							? getContextScrollTop( `#${ hash }`, breakpoint )
-							: 0,
-					behavior: 'smooth',
-				} );
-			} else {
-				global.history.replaceState(
-					{},
-					'',
-					removeQueryArgs( global.location.href )
-				);
-			}
-			setSelectedIds( selections );
+		( { target } ) => {
+			const chip = target.classList.contains( 'mdc-chip' )
+				? target
+				: target.parentNode;
```

Comment:
This will select the wrong chip if the click happens on an inner element of the svg resulting in Traffic being selected instead.

![image](https://user-images.githubusercontent.com/1621608/149307487-44932406-9f2e-44e8-96e0-6ce0e928bfc2.png)

```suggestion
			const chip = target.closest( '.mdc-chip' );
```

This works in my testing ^